pkg/action/action.go

```diff
@@ -244,3 +244,44 @@ func (c *Configuration) Init(getter genericclioptions.RESTClientGetter, namespac
 
 	return nil
 }
+
+func ConfigRequired(command string) bool {
+	var actionConfigRequired = map[string]bool{
+		// release commands
+		"get":         true,
+		"history":     true,
+		"install":     true,
+		"list":        true,
+		"releaseTest": true,
+		"rollback":    true,
+		"status":      true,
+		"template":    true,
+		"uninstall":   true,
+		"upgrade":     true,
+
+		//registry commands
+		"registry": true,
+		"chart":    true,
+
+		// chart commands
+		"create":     false,
+		"dependency": false,
+		"pull":       false,
+		"show":       false,
+		"lint":       false,
+		"package":    false,
+		"repo":       false,
+		"search":     false,
+		"verify":     false,
+		"completion": false,
+		"env":        false,
+		"plugin":     false,
+		"version":    false,
+		"docs":       false,
+
+		//root
+		"helm": false,
+	}
+
+	return actionConfigRequired[command]
```

Comment:
What happens in Go if the command parameter does not exist in the map?

This will be important as it seems highly likely to me that it will be forgotten sometimes to add a new command here.
A compiler failure would be best, which is not what will happen here.

What happens for plugins?

Reply:
Ah, true. I tested nonexistent subcommands, but not a subcommand that exists but hasn't been added to the map(or a plugin).

I will update this to check if the key exists in the map, and if it does not, return `true` by default. That way new commands and plugins will just have the current functionality of failing on invalid kubeconfigs until they are added to the map.

Initially I wanted to check if a command needed the config by some property in the `cobra.Command` struct, but that doesn't seem easy to do since the parameter indicating that it needs the action config is in the function that creates it(e.g. `newListCmd()`). Having something like that would expose this functionality to plugins as well, and remove the need for maintaining a map like this.